components/processor/gcomapiprocessor/processor.go

```diff
@@ -62,14 +63,13 @@ func newAPIProcessor(cfg *Config, settings component.TelemetrySettings, s signal
 		cl = gcom.NewMockGcomClient()
 	}
 
-	ic, err := cache.NewInstanceCache(
-		cache.InstanceCacheConfig{
+	ic, err := cache.NewMultiInstanceCache(
+		cache.InstanceCachesConfig{
 			CompleteCacheRefreshDuration:    cfg.Cache.CompleteRefreshDuration,
 			IncrementalCacheRefreshDuration: cfg.Cache.IncrementalRefreshDuration,
-			InstanceTypes:                   []client.InstanceType{client.Grafana},
+			GrafanaClusterFilters:           strings.Split(cfg.Cache.GrafanaClusterFilters, ","),
```

Comment:
that probably will account for whitespaces as well

Reply:
added `parseGrafanaClusterFilters` func to handle the parsing, case with spaces is handled there